recipes/libmount/meson/conandata.yml

```diff
@@ -0,0 +1,8 @@
+sources:
+  "2.40-rc2":
+    url: "https://github.com/util-linux/util-linux/archive/refs/tags/v2.40-rc2.tar.gz"
+    sha256: "0fdf4a47eb415639cfcdf2ca4cb76919ca971dc173ce5bb034857a77188a7c27"
+patches:
+  "2.40-rc2":
+    - patch_file: "patches/4ebdd569049ac48a3867a59537cea769fa319e8f.patch"
+    - patch_file: "patches/0001-meson-Only-build-blkzone-and-blkpr-if-the-required-l.patch"
```

Comment:
```suggestion
    - patch_file: "patches/0001-meson-Only-build-blkzone-and-blkpr-if-the-required-l.patch"
      patch_type: "bugfix"
      patch_source: "https://github.com/util-linux/util-linux/issues/2850"
```

Reply:
Ah, sorry about that. I left these patches in to test in CCI temporarily. Both have landed in the 2.40 branch, so they won't be necessary when 2.40 is properly released. I've got this marked as a draft until then.